internal/runnerinstall/ecs.go

```diff
@@ -0,0 +1,686 @@
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+
+		Run: func(ui terminal.UI) error {
+			runSvcArn, err = launchRunner(
+				ctx, ui, log, sess,
+				opts.AdvertiseClient.Env(),
+				executionRole,
+				taskRole,
+				logGroup,
+				i.Config.Region,
+				i.Config.CPU,
+				i.Config.Memory,
+				i.Config.RunnerImage,
+				i.Config.Cluster,
+				opts.Cookie,
+				opts.Id,
+				netInfo,
+				efsInfo,
+			)
+			return err
+		},
+
+		Cleanup: func(ui terminal.UI) error { return nil },
+	}
+
+	if err := lf.Execute(log, ui); err != nil {
+		return err
+	}
+
+	log.Debug("runner service started", "arn", *runSvcArn)
+
+	return nil
+}
+
+func (i *ECSRunnerInstaller) InstallFlags(set *flag.Set) {
+	set.StringVar(&flag.StringVar{
+		Name:   "ecs-region",
+		Usage:  "AWS region in which to install the Waypoint runner.",
+		Target: &i.Config.Region,
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-exeuction-role-name",
+		Target:  &i.Config.ExecutionRoleName,
+		Usage:   "The name of the execution task IAM Role to associate with the ECS Service.",
+		Default: "waypoint-runner-execution-role",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-task-role-name",
+		Target:  &i.Config.TaskRoleName,
+		Usage:   "IAM Execution Role to assign to the on-demand runner.",
+		Default: "waypoint-runner",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-cpu",
+		Target:  &i.Config.CPU,
+		Usage:   "The amount of CPU to allocate for the Waypoint runner task.",
+		Default: "512",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-memory",
+		Target:  &i.Config.Memory,
+		Usage:   "The amount of memory to allocate for the Waypoint runner task",
+		Default: "2048",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-runner-image",
+		Target:  &i.Config.RunnerImage,
+		Default: defaultRunnerImage,
+		Usage:   "The Waypoint runner Docker image.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:    "ecs-cluster",
+		Target:  &i.Config.Cluster,
+		Default: "waypoint-server",
+		Usage:   "The name of the ECS Cluster to install the Waypoint runner into.",
+	})
+
+	set.StringSliceVar(&flag.StringSliceVar{
+		Name:   "ecs-subnets",
+		Target: &i.Config.Subnets,
+		Usage:  "Subnets to install the Waypoint runner into.",
+	})
+}
+
+func (i *ECSRunnerInstaller) Uninstall(ctx context.Context, opts *InstallOpts) error {
+	ui := opts.UI
+	log := opts.Log
+
+	sg := ui.StepGroup()
+	defer sg.Wait()
+
+	s := sg.Add("Uninstalling Runner...")
+	defer func() { s.Abort() }()
+
+	sess, err := utils.GetSession(&utils.SessionConfig{
+		Region: i.Config.Region,
+		Logger: log,
+	})
+	if err != nil {
+		return err
+	}
+
+	// Find clusterArn which waypoint runner is installed into
+	ecsSvc := ecs.New(sess)
+	services, err := ecsSvc.DescribeServices(&ecs.DescribeServicesInput{
+		Cluster:  aws.String(i.Config.Cluster),
+		Services: []*string{aws.String("waypoint-runner-" + opts.Id)},
```

Comment:
Does this need to match the format of the other things? And why did we update the format on Docker and Nomad from this format to `waypoint-[opts.Id]-runner`?

Reply:
@krantzinator the naming convention for the runner was changed [after some feedback](https://github.com/hashicorp/waypoint/pull/3335#discussion_r900435044), but changed to append `-runner` to the end (instead of the ID) only after observing how the Helm chart was adding the suffix `-runner` too. 

That has been resolved for the k8s install, and Nomad/ECS can and should be made more consistent with the other platforms! Updating.